src/Lecture1.hs

```diff
@@ -63,7 +65,10 @@ sumOfSquares x y = error "TODO!"
 
 -}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
+lastDigit :: Integer -> Integer
+lastDigit n
+    | (n < 0) = ((-1) * n) `mod` 10
+    | otherwise = n `mod` 10
```

Comment:
That's correct! 💯 
And you also use the infix notation using backticks for `mod` 🔝 

You can simplify the implementation a bit using the standard function `abs` that returns an absolute value of a number:

```haskell
λ: abs 7
7
λ: abs (-7)
7 
```